index.css

```diff
@@ -103,6 +102,11 @@ li.PbenchQueItem::after {
    background-image: url(./images/WebServer.png);
  }
 
+ .div4{
```

Comment:
Doesn't "div4" seem like a fairly generic name for a very specific background pattern?